tests/ovm_test.go

```diff
@@ -0,0 +1,280 @@
+package tests
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/hex"
+	"io/ioutil"
+	"math/big"
+	"strings"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/rawdb"
+	"github.com/ethereum/go-ethereum/core/state"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/ethereum/go-ethereum/core/vm/runtime"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+var chainConfig params.ChainConfig
+
+func init() {
+	chainConfig = params.ChainConfig{
+		ChainID:             big.NewInt(1),
```

Comment:
Chain ID is not 108?

Reply:
Ah it technically doesn't break the tests to have the wrong chain ID here because ChainID is checked before txs are actually executed. (basically, we're not checking the EM chainID matches the Geth node ChainID)